src/components/input/Input.vue

```diff
@@ -101,8 +107,22 @@ export default {
                 return this.newValue
             },
             set(value) {
-                this.newValue = value
-                this.$emit('input', value)
+                /*If forcedSync - set localValue same as emited instantly(not default behaviour, for example v-model.trim changing value only on blur*/
+                if(this.forcedSync){
+                    if(this.trim)value=value.trim();
+                    else if(this.number)value = this.toNumber(value);
+                    //add your handlers and preparers here, for forced sync on conditional setters/getters inside parent computed/event's logic
+					// if(this.onlyIntegers){//allow input only integers
+						// if(!Number.isInteger(value))return;
+					// }
+					
+                }
+                if(this.prepare){//not related to v-model stuff
+                    value = this.prepare(value);
+                }
+                if(this.prepareValid && !this.prepareValid(value))return;
```

Comment:
Pre setter for local setters with condition's without syncOnBlur stuff, also for instant syncs. return true - to allow, false - to stop. It's before input coz we don't need to call input for setter, we can just make something like: this.computedValue = value and setter gonna be called.